aospy_synthetic/db/sqlalchemy/sqlalchemy_config.py

```diff
@@ -89,23 +92,29 @@ def __init__(self, session, AospyObj):
                     setattr(self, key,
                             self._get_db_obj(self._db_attrs[key]['db_cls'],
                                              session,
-                                             getattr(AospyObj,
-                                                     self._db_attrs[key]['obj'])))
+                                             sub_obj))
+
+    def __repr__(self):
+        repr = ''
+        for attr in self.__class__._metadata_attrs:
+            repr += '{}: {}\n'.format(attr, getattr(self, attr))
+            return repr
```

Comment:
It's possible (with the exception of perhaps logging) we may not need a ```repr``` for the DB objects, since in my mind the end user should not have access to them.

What do you think?